pkgs/development/python-modules/cherrypy/default.nix

```diff
@@ -7,29 +7,24 @@
 
 buildPythonPackage rec {
   pname = "cherrypy";
-  version = "18.1.2";
+  version = "18.3.0";
 
   disabled = !isPy3k;
 
   src = fetchPypi {
     pname = "CherryPy";
     inherit version;
-    sha256 = "1w3hpsg7q8shdmscmbqk00w90lcw3brary7wl1a56k5h7nx33pj8";
+    sha256 = "0q6cs4vrv0rwim4byxfizrlp4h6hmwg3n4baz0ga66vvgiz6hgk8";
   };
 
-  # Remove patches once 96b34df and 14c12d2
+  # Remove patches once 88d2163 and 713f672
```

Comment:
I only see 1 patch
```suggestion
  # Remove patches once 88d2163
```

Reply:
True. The [upstream PR](https://github.com/cherrypy/cherrypy/pull/1820) has two commits. One of them does not apply to the released version, because it reverts a change of a commit that happened after release, I think.

We can just drop or reword that line, to say something like "drop patches on next release", which tends to be self-evident anyways, because they fail to apply.